examples/seismic/plotting.py

```diff
@@ -14,13 +14,15 @@
 
 def plot_perturbation(model, model1, colorbar=True):
     """
-    Plot a two-dimensional velocity difference from two seismic :class:`Model`
+    Plot a two-dimensional velocity perturbation from two seismic `Model`
     objects.
 
-    :param model: :class:`Model` object of first velocity model.
-    :param model1: :class:`Model` object of the second velocity model.
-    :param source: Coordinates of the source point.
-    :param receiver: Coordinates of the receiver points.
+    Parameters
+    ----------
+    model : Model
+        object of first velocity model.
+    model1 : Model
+        object of the second velocity model.
```

Comment:
this should simply be:
"The second velocity model."

Reply:
and ``colorbar`` description missing

@mloubout

Reply:
```
    colorbar : boolean
        Option to plot the colorbar.
```